cross-dom-bridge-erc20/index.js

```diff
@@ -12,18 +12,18 @@ const l1Url = `https://eth-goerli.g.alchemy.com/v2/${process.env.GOERLI_ALCHEMY_
 const l2Url = `https://opt-goerli.g.alchemy.com/v2/${process.env.OPTIMISM_GOERLI_ALCHEMY_KEY}`
 
 
-// Contract addresses for DAI tokens, taken
-// from https://static.optimism.io/optimism.tokenlist.json
-const daiAddrs = {
-  l1Addr: "0x4f96fe3b7a6cf9725f59d353f723c1bdb64ca6aa",
-  l2Addr: "0xDA10009cBd5D07dd0CeCc66161FC93D7c9000da1"
-}    // daiAddrs
+// Contract addresses for OPTb tokens, taken
+// from https://github.com/ethereum-optimism/ethereum-optimism.github.io/blob/master/data/OUTb/data.json
+const erc20Addrs = {
```

Comment:
For your token you'll need to deploy your L2 token through our factory, more info here

Reply:
IDK. Just something explaining where these addresses came from and that you'd want to swap them out in prod? I know its just a tut but making it super clear would help.